Add scripts for test_connection_odbc on Windows

staticlibs opened the pull request:
`test_connection_odbc` requires the DSN setup either in an .ini file (on Linux and Mac) or in Windows registry.

This change adds Pyton scripts that can be used for DSN setup on Windows the same way as existing Bash scripts are used on Linux/Mac.

Testing: no functional changes, no new tests.

Fixes: #47

maiadegraaf reviewed (commented):
Thanks! Looks good to me.

Can you confirm that this works on your fork?

staticlibs commented:
@maiadegraaf 

Thanks for the review! Yes, I've checked locally that `test_connection_odbc.exe` is passing with this script. I also enabled it on Windows GH actions runs, which are passing too. Also, I've just realized that backslashes are not escaped in string literals (for some reason local python 3.9 did not complain on this, but the CI one did). I've added backslashes escaping and rebased the commit onto latest main now.

maiadegraaf commented:
Great thanks!
